README.rst

```diff
@@ -68,30 +69,33 @@ Mac/Linux
 
 .. code-block:: console
 
-    pip install virtualenv
-    virtualenv <your-env>
-    source <your-env>/bin/activate
-    <your-env>/bin/pip install google-cloud-storage
+    python -m venv env
+    source env/bin/activate
+    pip install google-cloud-storage
 
 
 Windows
 ^^^^^^^
 
 .. code-block:: console
 
-    pip install virtualenv
-    virtualenv <your-env>
-    <your-env>\Scripts\activate
-    <your-env>\Scripts\pip.exe install google-cloud-storage
+    py -m venv env
+    .\env\Scripts\activate
+    pip install google-cloud-storage
 
 
 Example Usage
 ~~~~~~~~~~~~~
 
 .. code:: python
 
+    # Imports the Google Cloud client library
     from google.cloud import storage
+
+    # Instantiates a client
     client = storage.Client()
+
+    # Creates a new bucket and uploads an object
     new_bucket = client.create_bucket('new-bucket-id')
     new_blob = new_bucket.blob('remote/path/storage.txt')
     new_blob.upload_from_filename(filename='/local/path.txt')
```

Comment:
Can we add a line to the very end of the quickstart doc? 

"Now that you've set up your Python client for Cloud Storage, you can get started running [Storage samples](https://github.com/googleapis/python-storage/blob/main/samples/README.md)." 

This just helps provide a next step.

Reply:
Love the idea! Added a next step to the end of the quickstart.